app/channels/room_channel.rb

```diff
@@ -9,8 +9,7 @@ def unsubscribed
   end
 
   def speak(message)
-    return unless current_user && current_user.admin?
-
+    return unless current_user&.admin?
```

Comment:
@Diptime Can you add a new line after line 12? Thanks!

Reply:
@Diptime please let us know if you are stuck in anyway here. Thanks

Reply:
I will go ahead and approve this and merge it seems the contributor may  help up atm and its not a major issue. Thanks all